k8s/model-service/service.yml

```diff
@@ -1,7 +1,9 @@
 apiVersion: v1
 kind: Service
 metadata:
-  name: model-service-svc
+  name: model-service
```

Comment:
Is there any particular reason for renaming this to `model-service`? The reason for appending `-svc` to it was to distinguish between possible other configurations that are also for `model-service`. Of course, there is still a label for it, but should be good practice to always have a dedicated name for each config file.

Reply:
This also makes it easier for when helm charts will be introduced as those will have variable names only for each component and will have the `kind` type appended (e.g. `-svc`, `-ingress`, etc)

Reply:
I moved the name to label but in that case I agree with you. Should I change the app label to `model-service` then or keep it as `model-service-svc`?

Reply:
The name should be `model-service-svc` and the label `model-service`.

Reply:
As reference, you can see how it's done for the app service file. There we have `app-svc` as the name and `app` as label